src/components/shared/snackNotification.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div id="ConnectionStatusSnack-ctn">
+    <v-snackbar v-model="visibility">
+      <span id="snack-message">
```

Comment:
@vickywane 
Maybe an easier way to handle this is to condition rendering on a computed visibility attribute (a function) instead of a fixes parameter. When the computed attribute is read by the v-model check, the function can return the current state and set it to false.

```
computed:
  visibility: () => { 
    if (this.isMessageNew) { 
      this.isMessageNew = false
     return true 
    } else {
     return false
    } 
```

then in `setConnectionStatusNotification` before the switch statement:

```
  this.isMessageNew = true
```